pkg/morph/client/notary.go

```diff
@@ -209,7 +210,12 @@ func (c *Client) depositNotary(amount fixedn.Fixed8, till int64) (res util.Uint2
 			zap.Int64("expire_at", till),
 			zap.Uint32("vub", vub),
 			zap.Error(err))
-		return util.Uint256{}, nil
+		return nil
+	}
+
+	_, err = c.rpcActor.WaitSuccess(txHash, vub, err)
+	if err != nil {
+		return fmt.Errorf("waiting for %s TX to be persisted: %w", txHash.Reverse(), err)
```

Comment:
```suggestion
		return fmt.Errorf("waiting for %s TX to be persisted: %w", txHash.StringLE(), err)
```
to clarify endianness

Reply:
made additional commit